render.sh

```diff
@@ -5,15 +5,25 @@ set -xe
 
 BUILD_REGISTRY="quay.io/redhat-user-workloads/ose-osc-tenant/trustee/"
 RELEASE_REGISTRY="registry.redhat.io/confidential-compute-attestation-tech-preview/"
+MIGRATE_PARAM="--migrate-level bundle-object-to-csv-metadata"
 
 for OCP_VERSION in v4.*
 do
     # A blank line in the log
     echo
     # Switch to the build registry, so `opm` can pull freely.
     sed -i "s|$RELEASE_REGISTRY|$BUILD_REGISTRY|" $OCP_VERSION/catalog-template.json
+
+    # enable migrate params for OCP 4.17 and onwards
+    VERSION=$(echo $OCP_VERSION | grep -o -E '[0-9.]+')
```

Comment:
a more specific variable name can help in telling the difference with `OCP_VERSION`, I suggest `OCP_VERSION_NUMERAL`